### PR TITLE
Content FieldTypeBinary to MUI ToggleButtons conversion

### PR DESCRIPTION
**Pull request by d88naimi**

### CODE REVIEW

**Comment by d88naimi** [comment]
Screen Shot 2022-05-24 at 2 03 57 PM

Screen Shot 2022-05-24 at 1 48 04 PM

File: src/apps/content-editor/src/app/components/Editor/Field/Field.js [hunk 1]
```diff
@@ -402,20 +410,49 @@ export default function Field({
...
-            label={FieldTypeLabel}
-            description={description}
-            tooltip={settings.tooltip}
-            required={required}
-            value={value}
-            onChange={onChange}
-            offValue={binaryFieldOpts[0]}
-            onValue={binaryFieldOpts[1]}
-          />
+          <>
```
Comment (shrunyan) [inline src/apps/content-editor/src/app/components/Editor/Field/Field.js:414]: This feels like a lot to have to write any time we need a `FieldTypeBinary`. @agalin920 should this get rolled up and provide a simpler interface?

Comment (shrunyan) [reply src/apps/content-editor/src/app/components/Editor/Field/Field.js:414]: Granted @d88naimi has 2 other PRs with this structure implemented. @d88naimi what's your time estimate on how long these 3 PRs took?

Comment (shrunyan) [reply src/apps/content-editor/src/app/components/Editor/Field/Field.js:414]: Something to consider. Even if it is quick to implement with some much structure and plumbing how much are the odds of something being done wrong increased?

Comment (d88naimi) [reply src/apps/content-editor/src/app/components/Editor/Field/Field.js:414]: @shrunyan all PRs took maybe 3 days with testing (10 FieldTypeBinary converted).

I estimate though now removing all the null checks and lifting that logic to a different location with the help of @agalin920 another 2-3 days though

**Review by agalin920 — APPROVED** [review]
Approved

**Pull request merged**